### DISCUSSION TITLE
One email address and 60 users

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#14907] [How to]
What would be the best way to let users send and receive correspondence to/from accounts/contacts?
The scenario is when there is only one company email set by Admin

**Comment by JohnMaverick** [comment]
Email configuration in yetiforce is unnecessarily complicated. I faced the same problems reported in other threads.
System should allow only Admins to manage email credentials. Every user signature should be integrated with user profile.
Roundcube slowing down and makes complicated all stuff. Integrating roundcube was more exchausting than writing from scratch (or porting from vtiger) native CRM email client.

**Reply by waran70** [reply]
Native app better.... can be simple. User want more, can buy integrations (if available)
